packages/core/src/__tests__/__e2e__/spinner/Spinner.accessibility.cy.tsx

```diff
@@ -33,31 +35,24 @@ describe("GIVEN an available announcer", () => {
 
   // TODO fix unmount announcement
   it.skip("THEN the announcer should be called when the component unmounts", () => {
-    cy.mount(<Spinner aria-label={ariaLabel} />).then(({ unmount }) => {
-      unmount().then(() => {
-        cy.get("body").should("announce", `finished ${ariaLabel}`);
-      });
-    });
+    cy.mount(<Spinner aria-label={ariaLabel} />);
+    cy.then(() => ReactDOM.unmountComponentAtNode(getContainerEl()));
+    cy.get("body").should("announce", `finished ${ariaLabel}`);
```

Comment:
was there a reason for these changes if the tests are still skipped?

Reply:
there were failing type checks